concent_api/core/views.py

```diff
@@ -261,15 +249,11 @@ def receive_out_of_band(request, _message):
         if last_undelivered_receive_status.type == 'AckReportComputedTask':
             serialized_ack_report_computed_task = last_undelivered_receive_status.data.tobytes()
 
-            try:
-                decoded_ack_report_computed_task = message.Message.deserialize(
-                    serialized_ack_report_computed_task,
-                    None,
-                    check_time = False
-                )
-
-            except InvalidSignature as exception:
-                return JsonResponse({'error': "Failed to decode AckReportComputedTask. {}".format(exception)}, status = 400)
+            decoded_ack_report_computed_task = message.Message.deserialize(
+                serialized_ack_report_computed_task,
+                None,
+                check_time = False
+            )
```

Comment:
This change is not "Change load() to deserialize() method to get data from database". You're just removing the exception handler that should have been removed in an earlier commit. It just adds noise and distracts from the relevant changes so it would be better as a separate cleanup commit.